### PR TITLE
Add S3 Support

### PR DESCRIPTION
**Pull request by jabbate19**
I did the thing!

### CODE REVIEW

**Comment by mxmeinhold** [comment]
Can you separate out your quote linting change?

File: deadass/templates/s3form.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{% extends "layout.html" %} {% block content %}
+<script type="text/javascript">
+  function create(this) {
+    document.user_form.action = "login.html";
+    alert(document.user_form.action);
```
Comment (Mstrodl) [inline deadass/templates/s3form.html:5]: Is this intended?

File: deadass/utils.py [hunk 2]
```diff
@@ -9,22 +9,24 @@ def wrapped_function(*args, **kwargs):
...
+        groups = session["userinfo"].get("groups", [])
         is_eboard = "eboard" in groups
         is_rtp = "rtp" in groups
         auth_dict = {
             "uid": uid,
             "first": first,
             "last": last,
             "picture": picture,
-            "admin": is_eboard or is_rtp or uid == "skyz"
+            "admin": is_eboard or is_rtp or uid == "skyz",
```
Comment (Mstrodl) [inline deadass/utils.py:20]: Consider removing this skyz bit?

**Review by Mstrodl — APPROVED** [review]
Lgtm, just take a look at these

**Pull request merged**